tests/storage/vmfactory.go

```diff
@@ -0,0 +1,48 @@
+/*
```

Comment:
Why do we need to put this in storage? I think DVs are general enough and used as part of other test suites